external_worker/local/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
```

Comment:
Let's keep it at `version: "3"` for broader compatibility. I believe there is nothing specific in this file to version `3.7`. See the main `docker-compose.yml` file in the root of this repo for reference